src/cljs/athens/views/blocks/core.cljs

```diff
@@ -255,13 +271,13 @@
                      (or (and (true? linked-ref) (:linked-ref/open @state))
                          (and (false? linked-ref) open)))
             (for [child children]
-              [:div {:key (:db/id child)}
+              [:<> {:key (:db/id child)}
                [block-el child
                 (assoc linked-ref-data :initial-open (contains? parent-uids (:block/uid child)))
                 opts]]))
 
-          [drop-area-indicator/drop-area-indicator #(when (= drag-target :below) {;; :color "red"
-                                                                                  :opacity "1"})]])))))
+          (when (= (:drag-target @state) :child) [drop-area-indicator/drop-area-indicator {:style {:grid-area "below"} :child true}])
+          (when (= (:drag-target @state) :below) [drop-area-indicator/drop-area-indicator {:style {:grid-area "below"}}])])))))
```

Comment:
Two drop indicators? `:child` drop indicator is no longer in `block-contents`?!? Nice!!!

Reply:
We could get away with a single drop-indicator if we wanted to be more clever about it, but I didn't think it was worth the complexity.